<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>429</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>455</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Student Management System</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ssl.qrc">
    <normaloff>:/smsht.png</normaloff>:/smsht.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>591</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Teachers</string>
     </attribute>
     <widget class="QPushButton" name="tLogin">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>150</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>150</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
     <widget class="QLabel" name="dbChecker">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>541</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>251</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Segoe UI Semibold</family>
           <pointsize>13</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Username:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="tUsername"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Segoe UI Semibold</family>
           <pointsize>13</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="tPassword">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Students</string>
     </attribute>
     <widget class="QPushButton" name="sLogin">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>150</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>150</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
     <widget class="QLabel" name="dbChecker_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>551</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>251</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Segoe UI Semibold</family>
           <pointsize>13</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Username:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="sUsername"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Segoe UI Semibold</family>
           <pointsize>13</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="sPassword">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Parents</string>
     </attribute>
     <widget class="QPushButton" name="pLogin">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>150</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>150</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
     <widget class="QLabel" name="dbChecker_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>551</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>251</width>
        <height>84</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Segoe UI Semibold</family>
           <pointsize>13</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Username:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Segoe UI Semibold</family>
           <pointsize>13</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="pPassword">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="pUsername"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>429</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tUsername</tabstop>
  <tabstop>tPassword</tabstop>
  <tabstop>tLogin</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>sUsername</tabstop>
  <tabstop>sPassword</tabstop>
  <tabstop>sLogin</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pUsername</tabstop>
  <tabstop>pPassword</tabstop>
  <tabstop>pLogin</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="ssl.qrc"/>
 </resources>
 <connections/>
</ui>
